### PR TITLE
Update team communication page re: IM

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/RoSbF1W0/450-document-gitter-irc-slack
This updates https://www.openmicroscopy.org/site/support/contributing-staging/team-communication.html to replace Jabber with Slack for everyday team communication and also mention that we don't provide support via IM (IRC or Gitter) although some team members do use IRC. 

(N.B. I took the IRC info from the Slack bridge, if someone who uses it wants to clarify re: used channels, please comment)

### CODE REVIEW

File: contributing/team-communication.txt [hunk 1]
```diff
@@ -11,44 +11,32 @@ find and track it may take some time.
...
+subgroup, or “mini group”, meetings are organized as needed, but should
+provide feedback (tickets, notes, etc). More information is available under
+:community_plone:`conference-calls <minutes/conference-calls>`.
+
+Other IM tools
+^^^^^^^^^^^^^^
+
+Slack is the only IM tool used by the entire OME team. Some team members do
+also use IRC (#ome on irc.freenode.net) but in general, there is no official
+support via IM and all external requests for help should be submitted and
```
Comment (joshmoore) [inline contributing/team-communication.txt]: This doesn't capture for me how this is being used. I think it is a _support_ channel, but all significant conversations must be captured in something more formal (trac, trello).

**Comment by hflynn** [comment]
Better now @joshmoore ?

**Comment by joshmoore** [comment]
> Better now @joshmoore ?

Yes, thanks. (but is `Openmicroscopy` a new spelling? :smile:)

**Comment by hflynn** [comment]
Got around the whole casing and URL vs name issue by just correcting to OME - okay to merge now @joshmoore ?

**Comment by joshmoore** [comment]
:+1:

**Pull request merged**